scipy-1.0/paper.tex

```diff
@@ -1156,25 +1157,17 @@ \section*{Discussion}
 
 \bibliography{references}
 
-\section*{Acknowledgments}
-
-\fixme{TODO}
-
-%Acknowledgements should be brief, and should not include thanks to anonymous
-%referees and editors, or effusive comments. Grant or contribution numbers may
-%be acknowledged.
-
 \section*{Author contributions statement}
 
-\fixme{TODO}
+M.H. edited the manuscript, created figures, compiled the author list and contributions statements, and drafted the \texttt{linprog} technical improvements section.
```

Comment:
So we're going to do this systematically then?

Reply:
If you're referring to the level of detail -systematically summarizing all contributions, I'm not sure. I thought I should give it a go so we could decide here.

If you are referring me compiling a list, it says in the template that all the authors have to be in the contributions statement. If they really mean that everyone's initials have to appear, I thought it would be easier for someone to collect and add them. I thought I'd just include it on the author form I made last year. Everyone who has contributed (for which an email address is available) will get an email with their PR/issue contributions, and they could summarize concisely?

Should probably include a bit about my contribution to SciPy itself rather than just the paper?

Reply:
I'd suggest to take a shortcut here, to avoid a large amount of work. What the AstroPy paper did: list separately all project maintainers, the people who did most of the writing, and all other contributors (on page 1). Then here you can simply say "group X did ..., group Y did ...".